src/test/java/cms/gov/madie/measure/services/ModelValidatorTest.java

```diff
@@ -328,4 +333,90 @@ void testQdmModelValidatorTestMeasureDoesHaveMeasuretypes() {
     assertTrue(validator instanceof QiCoreModelValidator);
     assertDoesNotThrow(() -> validator.validateGroups(measure));
   }
+
+  @Test
+  public void testValidateMetadataNoMetadata() {
+    Measure measure = Measure.builder().build();
+    ModelValidator validator = modelValidatorFactory.getModelValidator(ModelType.QI_CORE);
+    //    measureUtil.validateMetadata(measure);
+    assertDoesNotThrow(() -> validator.validateMetadata(measure));
+  }
+
+  @Test
+  public void testValidateMetadataNoDevelopers() {
+    Measure measure =
+        Measure.builder()
+            .id("measureId")
+            .measureMetaData(MeasureMetaData.builder().build())
+            .build();
+    ModelValidator validator = modelValidatorFactory.getModelValidator(ModelType.QI_CORE);
+
+    Exception ex =
+        Assertions.assertThrows(
```

Comment:
I think there are couple of codacy errors, one of them is this qualified usage as it already a static import.

Reply:
@adongare Ok thanks i'll take a look

Reply:
@adongare removed